tests/bsim/bluetooth/mesh/overlay_low_lat.conf

```diff
@@ -2,3 +2,4 @@ CONFIG_BT_EXT_ADV=n
 CONFIG_BT_CTLR_ADVANCED_FEATURES=y
 CONFIG_BT_CTLR_LOW_LAT=y
 CONFIG_BT_TICKER_LOW_LAT=y
+CONFIG_BT_MESH_FRIEND_ADV_LATENCY=2
```

Comment:
Could you elaborate why this is needed?

Reply:
I elaborated it here: https://github.com/zephyrproject-rtos/zephyr/pull/56423/files#diff-c40d529b65a93a6bf2872ca227ccec6ff4a2bf8eaa0b473f8ee397dd476e3955R544-R547

`mesh_msg` test that is run in almost all configurations and this is quite fragile to timing changes. After provisioning refactoring, the provisioning itself started working faster and test became broken even for tinycrypt on the legacy advertiser with a low latency feature. Adding more latency (so seems fair enough for a low latency option) it started passing again. In general, this test should be redesigned (what is in progress right now).